packages/ocs/dashboards/persistent-internal/activity-card/activity-card.tsx

```diff
@@ -167,9 +189,29 @@ const OngoingActivity = () => {
     });
   }
 
+  const isProviderMode = useFlag(PROVIDER_MODE);
+  const osdMigrationStatus = getOSDMigrationStatus(cephCluster);
+
+  if (
+    !isProviderMode &&
+    (osdMigrationStatus === OSDMigrationStatus.PENDING ||
+      osdMigrationStatus === OSDMigrationStatus.IN_PROGRESS)
+  ) {
+    resourceActivities.push({
+      resource: cephCluster,
+      timestamp: null,
+      loader: () =>
+        Promise.resolve(() => (
+          <ClusterMigrationActivity cephData={cephCluster} />
+        )),
```

Comment:
console should already be passing `resource` to the `loader`, check it once...
```suggestion
        Promise.resolve(ClusterMigrationActivity)
        )),
```